scraper.py

```diff
@@ -31,6 +31,8 @@
     if doctors_result:
         foreign_doctors_scraper.doc_id = len(doctors_result)
         foreign_docs_result = foreign_doctors_scraper.run_scraper()
+    else:  # in the event doctor scraper fails, attempt on foreign doctors with id 1
+        foreign_docs_result = foreign_doctors_scraper.run_scraper()
```

Comment:
this thought came about when I changed the url. They all come from the medical board so if one is down so will be the other. 👍